.cirrus.yml

```diff
@@ -182,6 +187,13 @@ task:
   env:
     CIRRUS_WORKING_DIR: "/tmp/flutter sdk"
     COCOAPODS_DISABLE_STATS: true
+    ANDROID_SDK_TOOLS_URL: https://dl.google.com/android/repository/sdk-tools-darwin-4333796.zip
+    ANDROID_SDK_ROOT: $HOME/Android
+    ANDROID_HOME: $HOME/Android
+    OPEN_JDK_URL: https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u202-b08/OpenJDK8U-jdk_x64_mac_hotspot_8u202b08.tar.gz
+    OPEN_JDK_REVISION: jdk8u202-b08
```

Comment:
You can probably use `mojave-flutter` image with Android SDK and Java pre-installed.

Reply:
The only thing I'm worried about with that is these images are meant to test the flutter SDK as well, and I'm worried it might cause confusion in the tests between which `flutter` is being tested